test/types/BigInt-type.spec.js

```diff
@@ -0,0 +1,14 @@
+/* global expect, BigInt */
+if (typeof BigInt === 'function') {
+  describe('BigInt type', () => {
+    it('should consider an instance equal to itself', () => {
+      const a = BigInt(1);
+      expect(a, 'to equal', a);
+    });
+
```

Comment:
It it worth adding a test alongside this that equality is not allowed against a number as a precaution?

Reply:
Good idea, added in 9a5c43ec